test/e2e/dsa/dsa.go

```diff
@@ -42,6 +58,24 @@ func describe() {
 	f := framework.NewDefaultFramework("dsaplugin")
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
 
+	ginkgo.BeforeEach(func() {
+		kubectl("node-feature-discovery", "apply", "-k", "deployments/nfd/kustomization.yaml")
```

Comment:
as commented in https://github.com/intel/intel-device-plugins-for-kubernetes/issues/1143#issuecomment-1277547731 it would probably be good to deploy NFD only once for all

Reply:
Is there any difference in this pr actually?
Only one It() exists.
Is there a case that we run all tests at once?

Reply:
@hj-johannes-lee 
> Is there a case that we run all tests at once?

We actually run them all at once

> Ah, well, maybe we do not need to copy and paste for all device plugins?

Solved, please take a look

Reply:
@ozhuraki I mean, do we run just "go test -v ./test/e2e/..." without "-args -ginkgo.focus <something>"? Is there a system that can test all plugins e2e tests? Sorry, maybe I was not clear.

But, anyway you changed since you saw it as better, so I believe it is better.! :)